examples/k3s.yaml

```diff
@@ -4,20 +4,22 @@
 # the ports are already forwarded automatically by lima:
 #
 # $ export KUBECONFIG=$PWD/kubeconfig.yaml
-# $ limactl shell k3s sudo cat /etc/rancher/k3s/k3s.yaml >$KUBECONFIG
+# $ limactl shell --workdir=/ k3s sudo cat /etc/rancher/k3s/k3s.yaml >$KUBECONFIG
```

Comment:
I assume you specify the `--workdir` to avoid the error message from the non-existing host `$PWD`. I've just created #40 to deal with this error message, and if it is accepted, then there is no need to add `--workdir` here as the actual directory doesn't matter for the command.

Reply:
Thanks!